bot/exts/evergreen/emoji_count.py

```diff
@@ -27,16 +28,20 @@ def embed_builder(self, emoji: dict) -> discord.Embed:
 
         if len(emoji) == 1:
             for key, value in emoji.items():
-                embed.description = f"There are **{len(value)}** emojis in the **{key}** category"
+                msg = f"There are **{len(value)}** emojis in the **{key}** category"
                 embed.set_thumbnail(url=random.choice(value).url)
+                embed.description = msg
         else:
             msg = ''
             for key, value in emoji.items():
                 emoji_choice = random.choice(value)
                 emoji_info = f'There are **{len(value)}** emojis in the **{key}** category\n'
-                msg += f'<:{emoji_choice.name}:{emoji_choice.id}> {emoji_info}'
+                if emoji_choice.animated:
+                    msg += f'<a:{emoji_choice.name}:{emoji_choice.id}> {emoji_info}'
+                else:
+                    msg += f'<:{emoji_choice.name}:{emoji_choice.id}> {emoji_info}'
             embed.description = msg
-        return embed
+        return embed, msg
```

Comment:
Your return typehint for this method no longer matches what you're returning.

```py
    def embed_builder(self, emoji: dict) -> discord.Embed:
```

Reply:
Anything else need fixing @scragly except this?